src/api/test-controller/execution-context.js

```diff
@@ -6,13 +6,18 @@ import exportableLib from '../exportable-lib';
 const OPTIONS_KEY = Symbol('options');
 
 function createRequire (filename) {
+    //Deprecated since: Node v12.2.0
     if (Module.createRequireFromPath)
         return Module.createRequireFromPath(filename);
 
-    const dummyModule = new Module(filename, module);
+    if (Module.createRequire)
+        return Module.createRequire(filename);
+
+    const dummyModule          = new Module(filename, module);
+    const nearbyModulesPath    = dirname(filename).concat('/node_modules');
 
     dummyModule.filename = filename;
-    dummyModule.paths    = [filename].concat(module.paths);
+    dummyModule.paths    = [filename, nearbyModulesPath].concat(module.paths);
```

Comment:
@AlexKamaev, could you please note why we put the filename in `paths`? Also, I think we should put not only `node_modules` from the current directory, but also include all parent `node_modules`: https://nodejs.org/api/modules.html#modules_loading_from_node_modules_folders

Reply:
I rechecked my code and concluded that it's not required to pass filename. @Dmitry-Ostashev 's changes look good to me